### ISSUE TITLE
Taxonomy for Database

### ISSUE THREAD

**Issue by dcdanko** [knights-lab/SHOGUN#20]
Not an issue with the program but looking for a bit of advice.

I'm building a BURST database based on refseq-microbial for use with SHOGUN. How did you build the example genome taxa file?

I'm guessing this is pretty straightforward.

Thank you!

https://github.com/knights-lab/SHOGUN/blob/master/shogun/tests/data/genomes.small.tax

**Comment by bhillmann** [comment]
Yes you can create a custom ```sheared_bayes.fixed.txt``` for a custom database. It isn't quite streamlined yet though. I should be able to add those updates soon. The scripts exists currently [here](https://github.com/knights-lab/analysis_SHOGUN/tree/master/scripts).

The steps are shear the database with ```shear_db.py```, align the sheared file to the database, and parse the b6 file with ```parse_shear.py```
